devlab/tests/migration_utils.py

```diff
@@ -0,0 +1,41 @@
+import os
+import yaml
+import config
+
+
+class MigrationUtils:
+    def __init__(self):
+        # TODO:
+        #  Using relative paths is a bad practice, unfortunately this is the
+        #  only way at this moment.
+        #  Should be fixed by implementing proper package module for Cloud Ferry.
+        self.main_folder = os.path.dirname(os.path.dirname(os.getcwd()))
+
+    def load_file(self):
+        file_name = 'configs/filter.yaml'
+        file_path = os.path.join(self.main_folder, file_name)
+        filtering = open(file_path, 'r')
+        filter_dict = yaml.load(filtering)
+        return [filter_dict, file_path]
+
+    def filter_objects(self, src_data_list, filtering_objects=None):
```

Comment:
1. Let's split into 2 methods: `filter_vms` and `filter_images`.
2. `filter_*` method must always work with arguments of a single type.